message FileItem
{
  required string filename = 2;
  required uint64 version = 3;
  required bytes  device_name = 4;
  required uint64 seq_no = 5;

  required bytes  file_hash = 6;
  required uint32 mtime = 7;
  required uint32 mode  = 8;
  required uint64 seg_num = 9;

  required uint32 is_complete = 10;
}
